.gitignore

```diff
@@ -8,3 +8,4 @@ vendor/
 /phpunit.xml
 /.phpcs-cache
 /phpstan.neon
+/.idea/
```

Comment:
I know, I just find it weird that we would have to use that in a library. To me the use case for this is targetting a production environment when you don't have a proper dev env that would use the same version of php. For a library, I don't see a use case. IMO this is kind of dangerous for instance if we have a 7.3 build, we will not be able to update our deps to versions that are no longer compatible with 7.2.13, and test it in realistic conditions. I also don't understand what this has to do with this PR.
I found 2 commits in this PR that touch that line. One says "cleanup based on comments", the other says "fix production version". But there is no explanation. Maybe it can be found in the comments, but I'm lazy, and so are people reviewing the git history. Don't make use browse github for an explanation, include everything that is necessary to understand your commits in the git log, please :pray:

Reply:
i've reverted this.

Reply:
Thank you!